modules/cnv_importing_vm_disk_to_pvc.adoc

```diff
@@ -7,6 +7,11 @@ controller. When a PVC is created with special
 short-lived import pod that attaches to the PV and downloads the virtual
 disk image into the PV.
 
+It is not currently possible to import images from insecure registries using
```

Comment:
I'd s/It is not currently possible to import images from insecure registries using/You cannot import images from insecure registries by using